frontend/src/lang/index.ts

```diff
@@ -5,7 +5,7 @@ import en from './modules/en';
 
 const i18n = createI18n({
     legacy: false,
-    locale: localStorage.getItem('lang') || 'zh',
+    locale: localStorage.getItem('lang') || 'en',
     fallbackLocale: 'en',
     globalInjection: true,
     messages: {
```

Comment:
The code difference is that the default language has been changed from Chinese ('zh') to English ('en'). No other significant differences or issues have been detected.